extensions/wikia/AdEngine/js/video/player/porvata/floater.js

```diff
@@ -0,0 +1,175 @@
+/*global define, require*/
+define('ext.wikia.adEngine.video.player.porvata.floater', [
+		'wikia.window',
+		'wikia.document'
+	], function (win, doc) {
+		'use strict';
+
+		var activeFloatingCssClass = 'floating',
+			compatibleSlots = ['TOP_LEADERBOARD'],
+			floatingVideoPadding = 28;
+
+		function resetAdContainer(top) {
```

Comment:
Works for the experiment but I wonder if we couldn't add or remove some classes of ours at the end. Worth checking what we're doing if a slot collapses.